references/firestore-facade/sf-firestore-facade-lookup-v1/sharedflowbundle/sharedflows/default.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<!--
+ Copyright 2023 Google LLC
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+<SharedFlow name="default">
+    <Step>
+        <Name>AM-SetFirestoreMock</Name>
+    </Step>
+    <Step>
+        <Condition>encodingType = "base64"</Condition>
+        <Name>AM-Base64EncodePathInfo</Name>
+    </Step>
+    <Step>
+        <Condition>flow.firestore.demo.enabled != "true"</Condition>
```

Comment:
For consistency: `mock` instead of `demo`?

Reply:
done